client/src/components/ExploreFilter.tsx

```diff
@@ -223,37 +90,34 @@ export const ExploreFilter = ({
 
   return (
     <div
-      className={classNames(
-        variant === 'mobile' ? 'mx-auto w-full' : 'ml-2 w-[90%]',
-        'm-2 flex h-fit flex-col flex-wrap rounded-lg bg-slate-50 dark:bg-neutral-800 dark:text-gray-200'
+      className={twMerge(
+        variant === 'mobile' ? 'w-full' : 'w-96',
+        'flex h-fit flex-col flex-wrap rounded-lg bg-slate-50 px-10 py-8 dark:bg-neutral-800 dark:text-gray-200'
       )}
     >
       <div className='flex flex-row'>
-        <h1 className='m-10 mb-2 text-2xl font-semibold'>Filter</h1>
+        <h1 className='mb-2 text-2xl font-semibold'>Filter</h1>
         <ResetButton
-          className='ml-auto mr-10 mt-10'
+          className='ml-auto'
           onClear={() => {
             setSelectedSubjects([]);
             setSelectedLevels([]);
             setSelectedTerms([]);
           }}
         />
       </div>
-      <div className='m-10 my-5'>
-        <div className='space-y-3'>
-          <h1 className='text-xl font-semibold'>Course Code</h1>
-          <InputBox
-            selected={selectedSubjects}
-            setSelected={(value) => setSelectedSubjects(value)}
-            options={codesOptions}
-          />
-        </div>
-        <SelectedCourseCodes
-          selected={selectedSubjects}
-          setSelected={setSelectedSubjects}
-        />
-        <div className='space-y-2'>
-          <h1 className='mt-3 text-xl font-semibold'>Level</h1>
+      <div className='py-2.5' />
+      <h1 className='text-xl font-semibold'>Course Code</h1>
+      <div className='py-1.5' />
+      {/* TODO: Clicking on an option with a query in the field is kinda broken*/}
```

Comment:
```suggestion
```

Is this still the case?

Reply:
Nope i fixed it just forgot to remove the comment